extensions/wikia/ArticleVideo/scripts/article-featured-video.js

```diff
@@ -61,7 +61,9 @@ require([
 			inAutoplayCountries = geo.isProperGeo(instantGlobals.wgArticleVideoAutoplayCountries),
 			inNextVideoAutoplayCountries = geo.isProperGeo(instantGlobals.wgArticleVideoNextVideoAutoplayCountries),
 			autoplayCookieName = 'featuredVideoAutoplay',
-			autoplay = cookies.get(autoplayCookieName) !== '0' && inAutoplayCountries;
+			autoplayEnabled = cookies.get(autoplayCookieName) !== '0' && inAutoplayCountries,
+			autoplay = autoplayEnabled && !document.hidden,
```

Comment:
We do `window.guaSetCustomDimension(37, autoplay ? 'Yes' : 'No');` so this change would pollute the stats.